packages/expo-contacts/android/src/main/java/expo/modules/contacts/ContactsModule.kt

```diff
@@ -127,7 +130,7 @@ class ContactsModule : Module() {
     get() = appContext.permissions ?: throw Exceptions.PermissionsModuleNotFound()
 
   private val currentActivity: Activity
-    get() = appContext.throwingActivity
```

Comment:
This was failing android builds with 
```
e: file:///Users/norsegaud/memlane-expo/node_modules/expo-contacts/android/src/main/java/expo/modules/contacts/ContactsModule.kt:133:24 Unresolved reference: throwingActivity
```

Reply:
this should stay as it was: https://github.com/expo/expo/blob/f24d4cd9dad2d22c7f1ac6fb2ab636056ff79c20/packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt#L362

If you don't have that value, you're contributing from an outdated fork (`throwingActivity` was added on Sept 13).

Reply:
I just realized this was because my expo version was older. I upgraded and did prebuild --clean to fix it.